network-area-diagram-viewer/src/main/java/com/powsybl/nad/viewer/JsHandler.java

```diff
@@ -0,0 +1,57 @@
+package com.powsybl.nad.viewer;
+
+import com.powsybl.iidm.network.Bus;
+import com.powsybl.iidm.network.Container;
+import com.powsybl.iidm.network.VoltageLevel;
+import com.powsybl.nad.model.Point;
+import javafx.beans.property.StringProperty;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class JsHandler {
+
+    private Model model;
+    private StringProperty modelSvgContent;
+    private Container<?> container;
+
+    private final Map<String, Point> postLayoutPositions = new HashMap<>();
```

Comment:
I don't think it's useful to separate positions from post-layout positions. We'd better have a `positions` map containing all positions. This map or its serialization should be the input of a `FixedPositionLayout` implementation of `Layout`, instead of the input of a force layout algorithm which does nothing as everything is already in place. 

We could say that we save only the minimum amount of information by saving only the changes, but then the saved position is linked with a specific layout algorithm. And if a user moves one node only and is satisfied with the result, and saves it, he wants all the nodes to be at the same place even if an updated algorithm has been released.

Reply:
Totally agree.

Reply:
I got rid of the post-layout positions. 
The update of the diagram is performed based on a complete set of positions, and it is done using a `FixedLayout` algorithm that does basically nothing except updating the nodes in the graph with the positions given as inputs.